docs/UserGuide.adoc

```diff
@@ -409,50 +410,84 @@ Format: `exit`
 TeachConnect data are saved in the hard disk automatically [even while sharing TeachConnect] after any command that changes the data. +
 There is no need to save manually.
 
-=== Importing the contacts : `import` `[since v1.2]`
+=== Importing the contacts : `import` `[since v1.4]`
 
 TeachConnect allows you to import contacts from a different TeachConnect file by specifying the location of the file. +
 
```

Comment:
TeachConnect allows you...
-->
Imports contacts from a different....
Consistency with other commands